src/scripts/clipperUI/componentBase.ts

```diff
@@ -67,7 +66,7 @@ export abstract class ComponentBase<TState, TProps> {
 	 * Example use:
 	 *      <a id="myCoolButton" {...this.enableInvoke(this.myButtonHandler, 0)}>Click Me</a>
 	 */
-	public enableInvoke(handleMethod: Function, tabIndex = 0, args?: any, idOverride: string = undefined) {
+	public enableInvoke(handleMethod: Function, tabIndex = 0, args?: any, idOverride: string = undefined, ariaSet: string = undefined) {
```

Comment:
Rename ariaSet to something that indicates that hints at directional key nav? ariaSet is not descriptive of what it's used for